resources/voting-stats.js

```diff
@@ -0,0 +1,112 @@
+let voteChart = null;
+let data = [];
+let min_possible_vote = 1;
+let max_possible_vote = 50;
```

Comment:
Why are these numbers hardcoded here?

Reply:
It should work fine if they are null.

Reply:
No I think he is referring to `min_possible_vote`, and `max_possible_vote`

Reply:
Yes, they can be null.